03-how/05-mangrove_friends.html.md

```diff
@@ -3,19 +3,22 @@ title: Mangrove Friends
 ```
 
 ### Applying to become a Mangrove friend
+
 Anyone can apply to join the Mangrove community using the application form on our website: https://bit.ly/joinMangroveCommunity
 
 ### Onboarding a Mangrove friend
-After several Mangrove members agreed on inviting a new person to the community because they share our values and have the ability to contribute (startup culture, skills & motivation), the first one to have given their agreement has to invite the applicant to fill the welcome form (find the link with the "newmf" command on Slack).
 
-Once they filled the form, the 1st member that gave their agreement will add the new Mangrove friend to the MF slack  groupand write an introduction in #general as soon as they join.
+After several Mangrove members agree on inviting a new person to the community because they share our values and have the ability to contribute (startup culture, skills & motivation), the first one to have given their agreement has to invite the applicant to fill the welcome form (find the link with the "newmf" command on Slack).
```

Comment:
Should we discuss the criteria to accept a MF and how the voting process works here? How many votes are needed to add a friend? When does this happen? Is there be a time every week where potential friends are discussed and voted on?